helpers/any.js

```diff
@@ -15,25 +15,37 @@ const factory = () => {
         const opts = args.pop();
         const predicate = opts.hash;
 
-        if (!_.isEmpty(predicate)) {
+        if (!common.isEmptyObject(predicate)) {
             // With options hash, we check the contents of first argument
-            any = _.any(args[0], predicate);
+            const predicateKey = Object.keys(predicate)[0];
```

Comment:
We only check the first key from the options hash? The lodash docs seem to imply otherwise: https://lodash.com/docs/2.4.2#some

Reply:
So this is another thing that has a disconnect with our docs (the other being pluck) where the example use is selected=true and we don't use more than one predicate in the cornerstone theme.